libraries/AP_HAL_ChibiOS/hwdef/HEEWING-F405/hwdef.dat

```diff
@@ -91,6 +91,10 @@ PA3 TIM9_CH2 TIM9 RCININT PULLDOWN
 
 # alternative with PA3 as USART2_RX
 PA3 USART2_RX USART2 ALT(1)
+DEFAULT_SERIAL6_PROTOCOL SerialProtocol_RCIN
+# ensure that the ALT config gets a DMA channel to support higher bitrate protocols (e.g. CRSF)
+define STM32_UART_USART2_RX_DMA_STREAM STM32_DMA_STREAM_ID(1, 5)
```

Comment:
this relies on a particular assignment, this makes it very fragile, we need another approach

Reply:
Actually it turns out it already works correctly on this board, so I am not sure under what circumstances the peripheral doesn't get DMA

Reply:
it works as USART2_TX is in the base config, only the RX is an ALT config